.changeset/empty-rockets-try.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': major
+---
+
+Added wrapperProps to allow for styling of CheckBox wrapper.
```

Comment:
Update changeset with all changes you've made

Reply:
Done